docs/src/lib/functions.md

```diff
@@ -4,10 +4,17 @@ CurrentModule = DataFrames
 
 # Functions
 
-## Multi-threading support
-
-Selected operations in DataFrames.jl automatically use multiple threads when available.
-It is task-based and implemented using the `@spawn` macro from Julia Base.
+## Multithreading support
+
+By default, selected operations in DataFrames.jl automatically use multiple threads
+when available. It is task-based and implemented using the `@spawn` macro from Julia Base.
+Multithreading can be disabled when running a particular block of code using the
+[`DataFrames.singlethreaded() do... end`(@ref) syntax. It can also be disabled
+globally by calling [`DataFrames.setmultithreading(true)`](@ref).
```

Comment:
```suggestion
globally by calling [`DataFrames.setmultithreading(false)`](@ref).
```
`DataFrames.setmultithreading(true)` sounds like enabling multithreading, not disabling it. Maybe I'm misunderstanding the sentence though.